src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1742,6 +1742,21 @@
                         </UserData>
                     </template>
 
+                    <template tid="ContentionStart_V2">
+                        <data name="ContentionFlags" inType="win:UInt8" map="ContentionFlagsMap" />
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="LockObjectID" inType="win:UInt64" outType="win:HexInt64" />
```

Comment:
`win:Pointer` would be a better data type for LockObjectID

Reply:
done https://github.com/dotnet/runtime/pull/72627/commits/5fb79fdff0d498b6e456d5a85bc9c9d97d5fb821